tox.ini

```diff
@@ -8,7 +8,7 @@ minversion = 3.10
 envlist =
     flake8
     mypy-{locked,lowest}
-    py{310,39,38,37,36}-{locked,lowest}
+    python{3.10,3.9,3.8,3.7}-{locked,lowest}
```

Comment:
:x:  it must be `py{310,39,38,37}`

Reply:
Interpreters are not found with the `py{310,39,38,37}`:

```
SKIPPED:  py39-locked: InterpreterNotFound: python3.9
SKIPPED:  py39-lowest: InterpreterNotFound: python3.9
SKIPPED:  py38-locked: InterpreterNotFound: python3.8
SKIPPED:  py38-lowest: InterpreterNotFound: python3.8
SKIPPED:  py37-locked: InterpreterNotFound: python3.7
SKIPPED:  py37-lowest: InterpreterNotFound: python3.7
```
I've changed as suggested here: https://github.com/tox-dev/tox/issues/378#issuecomment-413937359

Reply:
did you have python 3.7 to python 3.9 installed then you had tox tunning? 
tox is intended to skip interpreters that were not installed.

Reply:
please just revert it to the previous config. it was all as intended. 
just remove the `py36` from the list .


read https://tox.wiki/en/latest/config.html

Reply:
Yes you're right. I've missed the `skip_missing_interpreters` option there.